src/App.tsx

```diff
@@ -1,25 +1,131 @@
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import React, { useState } from 'react';
+
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import { Todo } from './types/Todo';
+
+function getUser(userId: number) {
+  return usersFromServer.find(user => user.id === userId) || null;
+}
+
+function getNewTodoId(todos: Todo[]) {
+  const maxId = Math.max(...todos.map(todo => todo.id));
+
+  return maxId + 1;
+}
+
+const todos: Todo[] = todosFromServer.map(todo => {
+  return {
+    ...todo,
+    user: getUser(todo.userId),
+  };
+});
+
+export const App: React.FC = () => {
+  const [initialTodo, setInitialTodo] = useState<Todo[]>(todos);
```

Comment:
```suggestion
  const [todos, setTodos] = useState<Todo[]>(todos);
```
The name is a bit confusing as it's an array of todos, not singular. Also, they are not initial, new todos will be added to this state too